docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,64 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
+* Add metrics for the execution time of filters and projections to `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`14135`)
+* Show local cost estimates when using `EXPLAIN`. ({issue}`14268`)
+* Fix timeouts happening too early because of improper handling of the
+  `node-scheduler.allowed-no-matching-node-period` configuration property. ({issue}`14256`)
+* Fix query failure for `MERGE` queries when `task_writer_count` is greater
+  than one. ({issue}`14306`)
+
+## CLI
+
+* Add display for the results of `EXPLAIN ANALYZE`. ({issue}`13907`)
+
+## Accumulo connector
+
+* Add support for column comments when creating a new table. ({issue}`14114`)
+* Add column mapping and index information in an extra field instead of a
+  comment. ({issue}`14095`)
+
+## BigQuery connector
+
+* Fix SQL injection vulnerability. ({issue}`14254`)
```

Comment:
do we need to link to a CVE or something like that?

Reply:
I don't think we have evidence this was exploited, it's just that unescaped slashes could've caused problems.

Reply:
So do we need to tone it down somehow?